lib/LedgerSMB/Sysconfig.pm

```diff
@@ -504,6 +520,27 @@ sub check_permissions {
     }
 }
 
+# if you have latex installed set to 1
+###TODO-LOCALIZE-DOLLAR-AT
+our $latex = 0;
+
+
+sub override_defaults {
+    # Check Latex
+    $latex = eval {require Template::Plugin::Latex; 1;};
+
+    # Check availability and loadability
+    $LedgerSMB::Sysconfig::template_latex = eval {require LedgerSMB::Template::LaTeX; 1}
```

Comment:
We should do a version check here and raise a sane error (on the console) if the installed version is too old.
Required version is listed in cpanfile but we can't obtain that directly as cpanfile is not guaranteed to be present (eg: system package install)